### PR TITLE
Update README to match Visual Studio Code's readme

### PR DESCRIPTION
**Pull request by boaz-codota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,73 @@
...
+<details><summary>Will TabNine leak my source code?</summary>
+<p>By default, TabNine makes web requests only for the purposes of downloading updates and validating registration keys. In this case your code is not sent anywhere, even to TabNine servers.
+You may opt in to TabNine Deep Cloud, which allows you to use TabNine’s servers for GPU-accelerated completions powered by a deep learning model. If sending code to a cloud service is not possible, we also offer a self-hosted option. 
+Contact us at enterprise@tabnine.com.
+</p>
+</details>
+
+### **Note:**
+___
+A note on licensing: this repo includes source code as well as packaged TabNine binaries. The MIT license only applies to the source code, not the binaries. The binaries are covered by the [TabNine End User License Agreement](https://tabnine.com/eula).
```
Comment (dimacodota) [inline README.md:73]: we need to change this, the link should be to the new terms of use page

**Review by dimacodota — CHANGES_REQUESTED** [review]
see my comments

**Review by dimacodota — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,15 +1,73 @@
...
+This is the Sublime Text client for [TabNine](https://tabnine.com), advanced AI based autocomplete for all programming languages. TabNine Indexes your entire project by reading your `.gitignore` [or others](https://www.tabnine.com/faq#nongit), and determines which files to index. 
+
+- Enter long variable names using  minimal keystrokes and leverage a mnemonic completion engine.
 - Zero configuration. TabNine works out of the box.
-- Highly responsive: typically produces a list of suggestions in less than 10 milliseconds.
+- Highly responsive: Produces a list of suggestions in less than 10 milliseconds on average.
+
+TabNine is part of [Codota](https://www.codota.com/) 
+
+<img src="https://raw.githubusercontent.com/codota/tabnine-vscode/master/assets/tabnine.gif" height="300" style="border-radius: 4px" />
```
Comment (amircodota) [inline README.md:15]: change url?

**Review by amircodota — APPROVED** [review]
LGTM. Had a minor comment

File: README.md (hunk 2 above)
Comment (boaz-codota) [reply README.md:15]: I've put this url specific to point to the vscode image. Would you prefer that I copy the gif to this repo?

File: README.md (hunk 1 above)
Comment (boaz-codota) [reply README.md:73]: Will be fixed in the future versions, that will include different messaging and SEO optimizations

**Pull request merged**